.goreleaser.yml

```diff
@@ -24,17 +33,28 @@ archives:
 - format: zip
   name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
 checksum:
+  extra_files:
+    - glob: 'terraform-registry-manifest.json'
```

Comment:
We would need to add this file to the root of the repository.

https://www.terraform.io/registry/providers/publishing#terraform-registry-manifest-file

> This file is conventionally named terraform-registry-manifest.json at the root of the provider repository and will be renamed in the release assets.

Reply:
I added the file.